spring-data-relational/src/main/java/org/springframework/data/relational/core/dialect/SQLiteDialect.java

```diff
@@ -0,0 +1,96 @@
+package org.springframework.data.relational.core.dialect;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.convert.ReadingConverter;
+import org.springframework.data.convert.WritingConverter;
+import org.springframework.data.relational.core.sql.LockOptions;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * SQL Dialect for SQLite.
+ *
+ * @author Rudolf Schmidt
+ * @since 3.3.0
+ */
+public class SQLiteDialect extends AbstractDialect {
+
+	public static final SQLiteDialect INSTANCE = new SQLiteDialect();
+
+	private SQLiteDialect() {
+	}
+
+	@Override
+	public LimitClause limit() {
+		return new LimitClause() {
+			@Override
+			public String getLimit(final long limit) {
```

Comment:
Nit: We do not use `final` on parameters. See also the other occurences.

Reply:
ok no problem I will remove it